### PR TITLE
Cleanup analysis

### PR DESCRIPTION
**Pull request by davkean**

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/DependenciesProjectTreeProvider.cs [hunk 1]
```diff
@@ -439,7 +439,7 @@ private async Task OnDependenciesSnapshotChangedAsync(SnapshotChangedEventArgs e
 
                 if (!cancellationToken.IsCancellationRequested)
                 {
-                    dependenciesNode = await viewProvider.BuildTreeAsync(dependenciesNode, snapshot, cancellationToken);
+                    dependenciesNode = await viewProvider!.BuildTreeAsync(dependenciesNode, snapshot, cancellationToken);
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/DependenciesProjectTreeProvider.cs]: https://github.com/dotnet/roslyn/issues/38548

Comment (drewnoakes) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/DependenciesProjectTreeProvider.cs]: This went away, then came back. It only seems to happen in the IDE, not as a build warning too. Hence I've been ignoring it.

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/DependenciesProjectTreeProvider.cs]: Build is a fixed version, IDE uses latest version,.

Comment (drewnoakes) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/DependenciesProjectTreeProvider.cs]: So yeah, it's a regression in Roslyn.

**Pull request merged**